src/databricks/labs/ucx/source_code/notebook.py

```diff
@@ -171,7 +169,7 @@ def build_dependency_graph(self, parent: DependencyGraph, problem_collector: Cal
                 return
         start_line = self._original_offset + 1
         problem = DependencyProblem(
-            'dependency-check',
+            'invalid-%run-cell',
```

Comment:
```suggestion
            'invalid-run-cell',
```
needs to be alphanumeric